oauth-function-template/datastore/table.js

```diff
@@ -57,7 +57,7 @@ const getOAuthTableColumnsToCreate = (ctx, tableId) => {
       switch (columnsResult.length) {
         case 0:
           return queries.tableColumns;
-        case 3:
+        case 4:
```

Comment:
Would be fantastic to have here `TABLE_COLUMNS.length` so we do not have to change it every time new column is added to OAuth Table